src/core/os/amdgpu/dri3/dri3WindowSystem.cpp

```diff
@@ -1116,6 +1116,83 @@ Result Dri3WindowSystem::GetOutputFromConnector(
     return result;
 }
 
+Result Dri3WindowSystem::FindCrtcForOutput(
```

Comment:
Functions should be marked with a // === header and a comment describing their purpose.  A blank comment may be acceptable for an obvious function (c'tor, init(), etc), but that's not this case.